posydon/binary_evol/MESA/step_mesa.py

```diff
@@ -1240,6 +1240,14 @@ def __init__(self, metallicity=1., grid_name=None, *args, **kwargs):
                          *args, **kwargs)
         # special stuff for my step goes here
         # If nothing to do, no init necessary
+        
+        # load grid boundaries
+        self.m1_min = min(self._psyTrackInterp.grid.initial_values['star_1_mass'])
+        self.m1_max = max(self._psyTrackInterp.grid.initial_values['star_1_mass'])
+        self.q_min = 0.05 # can be computed m2_min/m1_min
```

Comment:
Do we need a fixed value here? Because `q_min` will be a function depending on `m1`. Hence, I'm wondering, where it would be better to not specify a `q_min`, but instead provide a function `q_min(m1)`?

Reply:
You are right, about what we have in reality. Is a mass range for m1 and one for m2=m1*q at 0.5 < m2/Msun <  m1_max*0.99. Do you think is better to transform the q condition to a m2 condition?  Else I could add the extra condtion m2 = m1*q > 0.5 Msun which is the cut we add in practice.

Reply:
Yes, the `m2 = m1*q > 0.5 Msun` was, what I had in mind. Just to prevent you from doing a mistake here, there is still no global q_min, which you could derive from m1_min and m1_max, because we need to get a larger q_min in case m2 would get below 0.5Msun.